pods/testcmd/main.go

```diff
@@ -3,22 +3,26 @@
 package main
```

Comment:
is this just a manual test script? it could be added as a new subcommand to kne_cli if it adds functionality that users might want